_posts/2018-04-01-kudos-mapping-and-slam.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Kudos Mapping and SLAM
+date: 2018-04-01
+categories: 
+    - Robotics
+    - SLAM
+    - ROS
+    - Mapping
+description: Added LiDAR mapping functionality to Computer Science House's floor robot, Kudos.  
+image: https://github.com/ComputerScienceHouse/Kudos/blob/master/kudos_csh/images/CSH_Map_2.png?raw=true
+image-sm: http://www.trevorsherrard.com/images/CSH_Map_2.png
+author: Trevor Sherrard
+author-image: http://www.trevorsherrard.com/images/profile_pics.jpg
+author-bio: Electrical Engineer, roboticist, computer vision enthusiast. I have a strong passion seeing my software come alive in my projects.
```

Comment:
I know this has been touched on elsewhere, but I think "I have a strong passion *for* seeing my software come alive in my projects" would work better across the board.